### ISSUE TITLE
Documentation : Serial protocol

### ISSUE THREAD

**Issue by elliotwoods** [collin80/ESP32RET#5]
Hi there!

Thanks for sharing you great contribution!

I'm wondering if you have documented your serial protocol somewhere.
It seems you're keeping the protocol the same between your different RET device implementations.
I can read through the source code and figure it out piece by piece, but I'm wondering if there's a basic reference for the protocol already. 

Thank you!
Elliot

**Comment by collin80** [comment]
Yes, the best reference right now is here: https://github.com/collin80/M2RET/blob/master/CommProtocol.txt

It might not be 100% complete but should be pretty accurate.
